### ISSUE TITLE
[bug] Generative objkt has scrollbars

### ISSUE THREAD

**Issue by LeonNicholls** [teia-community/teia-ui#416]
### Describe the bug

https://teia.art/objkt/852386 displays vertical and horizontal scrollbars.

### Expected behavior

Not sure if this is expected, but I haven't seen this before.

### Platform and versions

```shell
Ubuntu 22.04
Chrome Version 125.0.6422.60 (Official Build) (64-bit)
```

**Comment by Zir0h** [comment]
This was also the case on the new text mints, maybe it needs the same fix

**Comment by melMass** [comment]
I'm sure it's "normal" since the dimensions are hardcoded.

Here is the full screen on my vertical monitor:
![image](https://github.com/teia-community/teia-ui/assets/7041726/3a369916-f690-4859-907c-dd606fbf39cd)

**Comment by Zir0h** [comment]
So it's a "bug/feature" of the token, it just doesn't want to scale?

**Comment by melMass** [comment]
Yes it uses hardcoded dimensions I'm not sure we can sanely handle cases like that
